Sources_v3/WebSocketClient/WebSocketClient.swift

```diff
@@ -88,6 +88,9 @@ class WebSocketClient {
         return pingController
     }()
     
+    /// The observer used to listen to `HealthCheckEvent`
+    private var healthCheckEventObserver: HealthCheckEventObserver?
```

Comment:
Finally figured out that we need either **health-check event observer** OR a **completion** in `EventNotificationCenter.process`.

The reason is **HealthCheckEvent** should also be processed via middlewares (especially by `EventDataProcessorMiddleware`) as it contains current-user payload that should be stored in a DB.

Decided to go with the observer as we don't need a completion anywhere else.